hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterDataSerializerHook.java

```diff
@@ -66,7 +68,7 @@
     public static final int CONFIG_MISMATCH = 11;
     public static final int GROUP_MISMATCH = 12;
     public static final int JOIN_CHECK = 13;
-    public static final int JOIN_REQUEST = 14;
+    public static final int JOIN_REQUEST_OP = 14;
```

Comment:
Renamed to disambiguate `JoinRequestOperation` vs `JoinRequest` class added below.